Story/RainMeadow.StoryHooks.cs

```diff
@@ -57,7 +57,33 @@ private void StoryHooks()
             On.SLOracleSwarmer.BitByPlayer += SLOracleSwarmer_BitByPlayer;
             On.CoralBrain.CoralNeuronSystem.PlaceSwarmers += OnCoralNeuronSystem_PlaceSwarmers;
             On.SSOracleSwarmer.NewRoom += SSOracleSwarmer_NewRoom;
+            On.HUD.TextPrompt.Update += TextPrompt_Update;
+        }
+
+        private void TextPrompt_Update(On.HUD.TextPrompt.orig_Update orig, TextPrompt self)
+        {
 
+            if (OnlineManager.lobby == null)
+            {
+                orig(self);
+                return;
+            }
```

Comment:
early returns look pretty but make logic hard to track.

right now if an online gamemode that isn't storygamemode would try and use this nothing would run (and I'd say that's unexpected)

can you rework so that you're only altering the behavior for storygamemode?